hostfactory/install.sh

```diff
@@ -43,7 +43,7 @@ function Generate-Provider-Config {
             \"name\": \"azurecc\",
             \"enabled\": 1,
             \"plugin\": \"azurecc\",
-            \"confPath\": \"${HF_CONFDIR}/providers/azurecc\",
+            \"confPath\": \"${HF_CONFDIR}/providers/azurecc/conf\",
```

Comment:
Again, let's check the other providers to see if they also stick an extra **conf** on at the end of the path :)